### PR TITLE
Change the coding style to PEP8

### PR DESCRIPTION
**Pull request by NamamiShanker**
## Related Issue
It is not related to any issue

## Describe the changes you've made
The code looked a little bit messy and difficult to look at, so I changed the style to PEP8 for a better understandability for future contributors.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: src/arguments/search.py [hunk 1]
```diff
@@ -74,21 +75,23 @@ def search_for_results(self, save=False):
...
             # evoke an error
-            search_error = SearchError("\U0001F613 No answer found", "\U0001F604 Please try reddit")
+            SearchError("\U0001F613 No answer found",
+                        "\U0001F604 Please try reddit")
         else:
             data = self.utility_object.get_ans(questions)
             print('''
-            \U0001F604 Hopefully you found what you were looking for! 
-            \U0001F4C2 You can save an answer to a file with '-file'
+            \U0001F604  Hopefully you found what you were looking for!
```
Comment (guptaaastha) [inline src/arguments/search.py:83]: There seems to be extra an whitespace here, please remove it to keep all interactive emoji dialogues consistent.

**Review by guptaaastha — CHANGES_REQUESTED** [review]
Please have a look at the inline comment, rest looks good to me!

File: src/arguments/search.py (hunk 1 above)
Comment (NamamiShanker) [reply src/arguments/search.py:83]: Oh right! I'll change it.

**Comment by NamamiShanker** [comment]
@pranavbaburaj I am not able to fix this one warning Codacy is throwing. I have tried whatever it is recommending. Do you have any ideas?

File: src/arguments/search.py (hunk 1 above)
Comment (NamamiShanker) [reply src/arguments/search.py:83]: This is a tab space. It helps in aligning text in the terminal. It aligns multiline suggestion "Open browser and post your question..."
image

Should I change it?

Comment (guptaaastha) [reply src/arguments/search.py:83]: If it's consistent throughout printing the dialogues, it's totally fine 😄

**Comment by pranavbaburaj** [comment]
> @pranavbaburaj I am not able to fix this one warning Codacy is throwing. I have tried whatever it is recommending. Do you have any ideas?

I have updated some files. All tests passed

**Review by guptaaastha — APPROVED** [review]
looks good to me

**Comment by anshsrtv** [comment]
Just remove the code smell. Everything else seems fine.

**Review by anshsrtv — COMMENTED** [review]
Just remove the code smell. Everything else seems fine.

**Pull request merged**